### ISSUE TITLE
Accessibility, Persona: Missing alt text and no props to add that

### ISSUE THREAD

**Issue by ipip2005** [OfficeDev/office-ui-fabric-react#1974]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: all

#### Priorities and help requested (not applicable if asking question):

Requested priority: (Normal) 

Products/sites affected: People web part

#### Describe the issue:

In persona control, the image element has
1. no alt text.
2. no way to add alt text to describe it to blind people.
3. no way to add "aria-hidden=true" to remove it from accessibility tree.

#### Actual behavior:

In page, the control is "visible" to screen readers and screen readers will just read "image" without more details which confuses blind people.

#### Expected behavior:

Developer can provide a way to specify the alt text of this people image. If no alt text is provided, it should has "aria-hidden=true"

**Comment by swese44** [comment]
Adding imageAlt prop and defaulting the image alt text to an empty string
https://github.com/OfficeDev/office-ui-fabric-react/pull/2065
